apps/system/style/app_titlebar.css

```diff
@@ -26,12 +27,28 @@
   height: .2rem;
 }
 
-.appWindow .titlebar .titlebar-maximized {
+.appWindow .titlebar {
+  opacity: 0;
+  visibility: hidden;
```

Comment:
why both opacity and visibility? are we transitioning/animating it sometimes?
if not visibility should be enough